include/invert_quda.h

```diff
@@ -245,6 +245,37 @@ namespace quda {
         interface)*/
     bool mg_instance;
 
+    /** The diagonal constant to suppress the low modes when performing 5D transfer */
```

Comment:
There's a lot of parameters here specifically for MADWF.  It would make it cleaner if we have a container that packs all these in.

Reply:
https://github.com/lattice/quda/pull/1061/commits/e5cf6f4989bb7a3c71a0d6b1d11b9348b8afdf38